packages/sdk-v2/src/sdk/claim/react.ts

```diff
@@ -175,6 +175,7 @@ export const useWhitelistSync = () => {
     };
 
     whitelistSync().catch(noop);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why not add proper deps ?
if you have issue with both [value, setValue] needed causing infinite effect indication, you could omit it by passing update callback to setter e.g.

```javascript
setValue(value => <calculate new value from value>)
```

OR/AND using refs for some flag-like props (e.g. syncInProgress). Just do not forgot to update those refs values via separate effect

Reply:
The rule has been removed, so these comments are not necessary anymore.
But after some testing and moving baseEnv to a ref I enabled the rule and the warning disappeared altogether

Also fixed a logic-error when its trying to find the backend url